swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, condition=None):
+        self.category = "Clothing"
+        self.condition = condition
+        self.message = "The finest clothing you could wear."
```

Comment:
In keeping with the learning goals, let's consider using standard conventions: `super` constructor. While this project can be done without it, and it may seem like a lot of overhead for such a small project, we should practice using the `super` constructor.

In real world scenarios, we want to lean on the power of OOP in Python to follow convention in the tech industry, but also to keep bugs at bay. The more we duplicate code, even something as small as retyping out `self.condition=condition`, can bring in bugs with misspells or unpredictable coding style.

I encourage you to consider refactoring your child classes with the `super` constructor for the practice!

```suggestion
    def __init__(self, condition=None):
        super().__init__("Clothing", condition)
        self.message = "The finest clothing you could wear."
```